src/pane-container.coffee

```diff
@@ -15,6 +15,8 @@ class PaneContainer extends Model
   @version: 1
 
   root: null
+  stoppedChangingActivePaneItemDelay: 100
```

Comment:
I think 100ms is a good starting point. We can adjust it once we start using this method in packages.